_data/team-members.yml

```
@@ -194,10 +194,12 @@ members:
     twitter: adrienpi2
     photo: /resources/img/adrien.png
     currentlyWorkingOn: true
-    contact_for: [scaladex]
+    contact_for: [scala3MigrationGuide]
```

```suggestion
    contact_for: [scaladex,scala3MigrationGuide]
```

If this is fine to @adpi2

---

done